provision/kubernetes/monitor.go

```diff
@@ -123,3 +140,79 @@ func (c *routerController) addPod(pod *apiv1.Pod) {
 		rebuild.EnqueueRoutesRebuild(appName)
 	}
 }
+
+func (p *clusterController) getPodInformer() (v1informers.PodInformer, error) {
```

Comment:
minor thing, but maybe replace p to c here and in the cases below, just to keep the pattern